app/scripts/lib/app-start.js

```diff
@@ -110,6 +111,7 @@ function (
   User,
   FormPrefill,
   Notifications,
+  PageRefreshWatch,
```

Comment:
> `BrowserRefreshWatch`

Hmm, mebbe. Or mebs just `RefreshWatch`? Lose the page/browser/whatever bit entirely?